js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -48,7 +48,7 @@ clientDescriptionHelp=Specifies description of the client. For example 'My Clien
 rolesPermissionsHint=Determines if fine grained permissions are enabled for managing this role.  Disabling will delete all current permissions that have been set up.
 passwordPoliciesHelp.regexPattern=Requires that the password matches one or more defined Java regular expression patterns.
 oAuthDPoP=OAuth 2.0 DPoP Bound Access Tokens Enabled
-invalidRealmName=Realm name can't contain special characters
```

Comment:
there are 2 scenarios that I tried locally:

- Entering a valid Realm ID: The realm gets created successfully
- Entering an invalid realm ID. For example "test$$". Here the backend call to POST /admin/realms returns a 404 with following error:
```
{
"error": "Character '$' not allowed.",
"error_description": "For more on this error consult the server log at the debug level."
}
```
But no invalidity error is shown. The behavior is the same on the main branch. So this is not a regression because of this change.